olwebsite-app/src/components/Toolbar/__tests__/HamburgerToolbar.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { render, fireEvent, getByLabelText } from '@testing-library/react';
+import HamburgerToolbar from '../HamburgerToolbar';
+import {  BrowserRouter as Router } from "react-router-dom";
+
+describe("Hamburger Toolbar Component Tests", function() {
+    let renderedComponent;
+
+    beforeEach(() => {
+        renderedComponent = render(<Router><HamburgerToolbar/></Router>);
+        global.innerWidth = 350;
+    });
+
+    test("initial render, dropdown should not be shown", () => {
+        const links = renderedComponent.container.querySelector('.link-dropdown');
+        expect(links).toBeNull();
```

Comment:
jest-dom/extend-expect is included in the test-utils.js file, so you've got access to convenient matchers like toBeInTheDocument and toBeVisible. You might want to use those just to be clearer.

Reply:
👍